scripts/zephyr_module.py

```diff
@@ -257,27 +257,32 @@ def process_blobs(module, meta):
     return blobs
 
 
-def kconfig_snippet(meta, path, kconfig_file=None):
+def kconfig_snippet(meta, path, kconfig_file=None, blobs=False):
     name = meta['name']
     name_sanitized = meta['name-sanitized']
 
-    snippet = (f'menu "{name} ({path.as_posix()})"',
+    snippet = [f'menu "{name} ({path.as_posix()})"',
                f'osource "{kconfig_file.resolve().as_posix()}"' if kconfig_file
                else f'osource "$(ZEPHYR_{name_sanitized.upper()}_KCONFIG)"',
                f'config ZEPHYR_{name_sanitized.upper()}_MODULE',
                '	bool',
                '	default y',
-               'endmenu\n')
+               'endmenu\n']
+
+    if blobs:
+        snippet.insert(-1, '	select TAINT_BLOBS')
     return '\n'.join(snippet)
```

Comment:
if doing the other proposed change, then this must be updated to:
```suggestion
    return ''.join(snippet)
```